### DISCUSSION TITLE
是否支持服务商分账

### DISCUSSION THREAD

**Discussion by o624366705** [w7corp/easywechat#2227] [Q&A]
超哥，请问是否支持服务商分账？

**Comment by overtrue** [comment]
貌似支持吧，只不过没有文档，你看看：https://github.com/w7corp/easywechat/blob/5.x/src/Payment/ProfitSharing/Client.php

**Reply by o624366705** [reply]
看了官方文档，服务商模式下分账与单商户分账唯一区别在于多了sub_appid，sub_mchid两个参数，接口地址和其它参数一致，目前easywechat看了源码是单商户分账，我继承一下client自己扩展，感谢！
